aci/resource_aci_lacpifpol.go

```diff
@@ -0,0 +1,209 @@
+package aci
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/ciscoecosystem/aci-go-client/client"
+	"github.com/ciscoecosystem/aci-go-client/models"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func resourceAciLACPMemberPolicy() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceAciLACPMemberPolicyCreate,
+		UpdateContext: resourceAciLACPMemberPolicyUpdate,
+		ReadContext:   resourceAciLACPMemberPolicyRead,
+		DeleteContext: resourceAciLACPMemberPolicyDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: resourceAciLACPMemberPolicyImport,
+		},
+
+		SchemaVersion: 1,
+		Schema: AppendAttrSchemas(map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"prio": {
```

Comment:
```suggestion
			"priority": {
```
Update all other references.

Reply:
ok